UnitsNet/UnitDefinitions/Energy.json

```diff
@@ -136,8 +136,8 @@
     {
       "SingularName": "ThermImperial",
       "PluralName": "ThermsImperial",
-      "FromUnitToBaseFunc": "x*1.05505585257348e+14",
-      "FromBaseToUnitFunc": "x/1.05505585257348e+14",
+      "FromUnitToBaseFunc": "x*1.05967e8",
```

Comment:
[Was wrong](http://www.wolframalpha.com/input/?i=1+therm_39+to+J)

Reply:
There are actually 4 definitions: Therm EC/US/UK and BTU

https://en.wikipedia.org/wiki/Therm#Definitions

ThermImperial may be misleading in name, but refers to Therm UK.
There is already a unit for BritishThermalUnit (BTU) further up here.

So I think it is already correct, as is. Do you agree?

Reply:
I did assume it was therm(UK). I thought that was therm_39, but looks like I'm completely wrong there. It caught my eye immediately though because the scale is completely off (e14).

If you look at Wikipedia, it should be 1.05505585257348e8, not 1.05505585257348e14. Looks like the decimal on Wikipedia got overlooked :)

So just to make sure I'm not missing anything, 1.05505585257348e8 seems correct, do you agree?

Reply:
Ah I missed that part :-D Yup, should be `e8`!

Reply:
I will change it to that then!